mainnet.json

```diff
@@ -4,115 +4,283 @@
             "name": "USD Coin",
             "ticker": "USDC",
             "address": "0xaf88d065e77c8cc2239327c5edb3a432268e5831",
-            "decimals": 6
+            "decimals": 6,
+            "supported_exchanges": {
+                "Binance": "USDC",
+                "Coinbase": "USD",
+                "Kraken": "USD",
+                "Okx": "USDC"
+            },
+            "chain_addresses": {
+                "1": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+                "1151111081099710": "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v"
+            },
+            "logo_url": "https://raw.githubusercontent.com/renegade-fi/token-mappings/refs/heads/main/token-logos/usdc.png"
         },
         {
             "name": "Tether USD",
             "ticker": "USDT",
             "address": "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9",
-            "decimals": 6
+            "decimals": 6,
+            "supported_exchanges": {
+                "Binance": "USDT",
+                "Coinbase": "USD",
+                "Kraken": "USD",
+                "Okx": "USDT"
+            }
```

Comment:
why not add a logo url here, even if we don't need it?

Reply:
yeah sure, let me find one & add it to this PR